### PR TITLE
Improvements done:

### PR DESCRIPTION
**Pull request by Naveendurai**

### CODE REVIEW

File: cvscan/data_operations.py [hunk 1]
```diff
@@ -255,3 +255,56 @@ def remove_extra(extra_info):
   with open(DATAPATH + 'extra/extra','wb') as fp:
     pickle.dump(extra, fp)
   logging.debug("extra file written")
+
+
```
Comment (lakshmanaram) [inline cvscan/data_operations.py]: Wouldn't it be better to have one utility function show returning the details depending upon the inputh path passed as a parameter.  
```python
def show(input_path):  
  with open(input_path,'rb') as fp:  
    required = pickle.load(fp)  
  print '::'.join(required)  
```

File: cvscan/details_parser.py [hunk 2]
```diff
@@ -80,10 +80,10 @@ def fetch_phone(resume_text):
 
 """
 def fetch_address(resume_text):
-  pincode_input_path = dirpath.PKGPATH + '/data/address/pincodes'
-  address_input_path = dirpath.PKGPATH + '/data/address/pincode-district-state'
-  states_input = dirpath.PKGPATH + '/data/address/states'
-  district_state_input = dirpath.PKGPATH + '/data/address/district-states'
+  pincode_input_path = dirpath.DATAPATH
```
Comment (lakshmanaram) [inline cvscan/details_parser.py]: This can result in broken links all over the file

Comment (lakshmanaram) [inline cvscan/details_parser.py]: Just now checked that this has been brought back in the future commits. Please do ignore :+1:

Comment (lakshmanaram) [inline cvscan/details_parser.py]: Again storing these file paths in the dirpath file as their respective variables can work out better.

Comment (swaathi) [inline cvscan/details_parser.py]: Yes. I hate creating new paths and not having control over it. :/

**Pull request merged**